environment.yml

```diff
@@ -0,0 +1,18 @@
+name: funconn
+channels:
+  - defaults
+dependencies:
+  - ca-certificates=2021.10.26=hecd8cb5_2
+  - cyrus-sasl=2.1.27=h8a25369_8
+  - icu=68.1=h23ab428_0
+  - krb5=1.19.2=hcd88c3b_0
+  - libcxx=12.0.0=h2f01273_0
+  - libdb=6.2.32=h0a44026_0
+  - libedit=3.1.20210910=hca72f7f_0
+  - libuv=1.40.0=haf1e3a3_0
+  - mysql=5.7.24=h56848d4_0
+  - ncurses=6.3=hca72f7f_2
+  - nodejs=14.8.0=hec0b97e_0
+  - openssl=1.1.1m=hca72f7f_0
+  - zlib=1.2.11=h4dc903c_4
+prefix: /Users/bdsinger/miniconda3/envs/funconn
```

Comment:
B. This hard-coded absolute path (`/Users/bdsinger/miniconda3/envs/funconn`) shouldn't be in the repo. Maybe this file should be in `.gitignore`?

Reply:
mistake, removed that file